Flixable_com_Scraper_.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>The long series of <code>try</code>/<code>except</code> blocks could be designed around to have more readable/maintainable code.  The issue issue being protected against is that <code>get<em>extra</em>info()</code> returns <code>None</code> when <code>response.ok</code>  is <code>False</code> ; a first pass idea might be to return a <code>defaultdict()</code>  that returns <code>np.nan</code>  (i.e. <code>defaultdict(lambda: np.nan)</code> this would handle all the try/excepts instead of the Ratings ones) example: <a href="https://gist.github.com/AdamSpannbauer/e73937864d50da900f11ea635b8e0802" target="_blank">https://gist.github.com/AdamSpannbauer/e73937864d50da900f11ea635b8e0802</a>).  Likely a more elegant solution to be found to handle all the cases though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AdamSpannbauer/flixable_ml_dsi/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>